RceG3/ruckus.tcl

```diff
@@ -3,13 +3,17 @@ source -quiet $::env(RUCKUS_DIR)/vivado_proc.tcl
 
 # Load local Source Code
 loadSource -dir  "$::DIR_PATH/hdl/"
+
 loadSource -path "$::DIR_PATH/simlink/rtl/RceG3CpuSim.vhd"
-loadIpCore -dir  "$::DIR_PATH/coregen/processing_system7_0/"
+
+loadIpCore -dir       "$::DIR_PATH/coregen/processing_system7_0/"
+loadBlockDesign -path "$::DIR_PATH/coregen/pcie_root/pcie_root.bd"
```

Comment:
Maybe we should move this loadBlockDesign to the DtmCore's ruckus.tcl because DPM doesn't would never use this